openedx/openedx-platform
# Cohorting API

This PR implements an alternative cohort management API for server-to-server communication.

**Sandbox URL**

LMS: https://pr18975.sandbox.opencraft.hosting/
Studio: https://studio-pr18975.sandbox.opencraft.hosting/

**Testing instructions**:

1. Create an Oauth client at http://localhost:18000/admin/oauth2/client if one does not exists for testing.
2. Use the Oauth2 endpoint to get an auth token (cUrl, Postman or other tool).
3. Check provided endpoints using the generated token, i.e. GET /api/cohorts/v1/courses/{course_id}/cohorts

**Reviewers**
